RELEASE-NOTES.adoc

```diff
@@ -19,6 +19,12 @@ endif::[]
 == Table of Contents
 toc::[]
 
+== v1.0.0-rc1
+
+=== Bug fixes
+
+* Fix EdgeLB pool name when 63 character limit is reached. The Kubernetes cluster name is now properly truncated to not hit the EdgeLB pool name 63 character limit.
```

Comment:
it sounds a bit repetitive `63 character limit` :) but I am not a native speaker here.

Reply:
What do you think now?

Reply:
sgtm, thx